test/otel_telemetry_SUITE.erl

```diff
@@ -51,7 +51,6 @@ telemetry_span_handling(_Config) ->
         error:badarg -> ok
     end,
     ?assertMatch(SpanCtx1, ?current_span_ctx),
-    ?set_attribute(<<"duration">>, 1),
```

Comment:
This should still be here as it simulates a user setting attributes on an open span, but the key name can be changed.

Reply:
Added a new attribute